Flow.Launcher/PublicAPIInstance.cs

```diff
@@ -114,6 +114,11 @@ public void ShellRun(string cmd, string filename = "cmd.exe")
             var startInfo = ShellCommand.SetProcessStartInfo(filename, arguments: args, createNoWindow: true);
             ShellCommand.Execute(startInfo);
         }
+        
+        public void CopyToClipboard(string text)
+        {
+            Clipboard.SetText(text);
```

Comment:
Pretty sure this method has been causing all kinds of grief back in the days, you want this instead:
https://github.com/Flow-Launcher/Flow.Launcher/blob/365f7e73c8ea8064eeedeced432a9ada8311d9d3/Plugins/Flow.Launcher.Plugin.Calculator/Main.cs#L83

Reply:
"Clipboard.SetDataObject() : This method attempts to set the data ten times in 100-millisecond intervals, and throws an ExternalException if all attempts are unsuccessful."

Interesting...